validation-test/stdlib/StringGraphemeBreaking.swift

```diff
@@ -102,7 +102,7 @@ extension _StringGuts {
   func _isForeign() -> Bool
 }
 
-if #available(SwiftStdlib 5.6, *) {
+if #available(SwiftStdlib 5.9, *) {
```

Comment:
Is `SwiftStdlib 5.9` availability defined on the 5.8 branch?

Reply:
It is now 😃